src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -256,14 +251,14 @@
             optional = true,
             doc = "Whether to ignore variants that have been filtered or to annotate them."
     )
-    protected boolean ignoreFilteredVariants = FuncotatorArgumentDefinitions.IGNORE_FILTERED_VARIANTS_DEFFAULT_VALUE;
+    protected boolean ignoreFilteredVariants = true;
```

Comment:
This value switched from false -> true didn't it?  Typically if we have a boolean commandline option we want to to always be false by default.  ItSo if you want the default to be true here it's probably best to switch it to "includeFilteredVariants" and invert it.

Reply:
Sounds good.  I switched it because my understanding is that this is the default operating mode.